Automattic/fresh-data
# Update/move reducer to framework

Pre-work for #203

The state reducer for fresh-data has become intrinsic to the core functionality of the framework and should always exist in a consistent state. Therefore this PR moves the reducer into the framework package so it can exist for any installation.

To Test:
Everything should work as normal, there should be no perceived differences in operation.
1. `npm install`, `npm run bootstrap`
2. Build and link framework: `cd packages/framework`, `npm install`, `npm run build`, `npm link`
3. Build and link provider: `cd packages/react-provider`, `npm install`, `npm link @fresh-data/framework`, `npm run build`, `npm link`, `rm -rf node_modules` (this dir interferes with npm link use)
4. Test each example: `cd examples/<example>`, `npm install`, `npm link @fresh-data/framework`, `npm link @fresh-data/react-provider`, `npm start` and then check normal operation.
